x-pack/qa/operator-privileges-tests/build.gradle

```diff
@@ -0,0 +1,38 @@
+import org.elasticsearch.gradle.info.BuildParams
```

Comment:
This should be in `x-pack/plugin/security/qa`

Reply:
Relocated. I initially choose `x-pack/qa` because it seems stuff related to "TestPlugin" implementation is all located there.